### ISSUE TITLE
Strange ssr behavior in v-for loop with if-else and v-text

### ISSUE THREAD

**Issue by TrueCarry** [vuejs/vue#11770]
### Version
2.6.11

### Reproduction link
[https://github.com/TrueCarry/linktest](https://github.com/TrueCarry/linktest)

### What is actually happening?

This is a duplicate of https://github.com/vuejs/vue/issues/1176, can we reopen it please?

**Comment by posva** [comment]
You seem to be referencing the wrong issue so it's hard to follow without a boiled down reproduction (no Nuxt). Open a new issue with a boiled down reproduction with only Vue. These docs should help: http://ssr.vuejs.org/
